filebeat/tests/system/test_crawler.py

```diff
@@ -761,6 +761,32 @@ def test_file_no_permission(self):
         """
         Checks that filebeat handles files without reading permission well
         """
+        if os.name == "nt":
+            # Currently skipping this test on windows as it requires `pip install win32api`
+            # which seems to have windows only dependencies.
+            # To solve this problem a requirements_windows.txt could be introduced which would
+            # then only be used on Windows.
+            #
+            # Below is some additional code to give some indication on how the implementation
+            # to remove permissions on Windows (where os.chmod isn't enough) could look like:
+            #
+            # from win32 import win32api
+            # import win32security
+            # import ntsecuritycon as con
+
+            # user, domain, type = win32security.LookupAccountName(
+            #     "", win32api.GetUserName())
+            # sd = win32security.GetFileSecurity(
+            #     testfile, win32security.DACL_SECURITY_INFORMATION)
+
+            # dacl = win32security.ACL()
+            # # Remove all access rights
+            # dacl.AddAccessAllowedAce(win32security.ACL_REVISION, 0, user)
+
+            # sd.SetSecurityDescriptorDacl(1, dacl, 0)
+            # win32security.SetFileSecurity(
+            #     testfile, win32security.DACL_SECURITY_INFORMATION, sd)
+            raise unittest.SkipTest
```

Comment:
QQ: is it possible to add a message stating why this test is being skipped? Something simple like "not supported on Windows"